kratos.gid/apps/Pfem/xml/GetFromXML.tcl

```diff
@@ -2,6 +2,7 @@ namespace eval Pfem::xml {
     variable dir
     variable bodyNodalCondition
     variable body_UN
+    variable Elements

```

Comment:
Be careful, I think this variable is unused but I think that this way is dangerous (trying to override core variables)

Reply:
I will delete it in another PR.